docs/crio.conf.5.md

```diff
@@ -280,12 +280,6 @@ CRI-O reads its configured registries defaults from the system wide containers-r
 **image_volumes**="mkdir"
   Controls how image volumes are handled. The valid values are mkdir, bind and ignore; the latter will ignore volumes entirely.
 
-**insecure_registries**=[]
-  List of registries to skip TLS verification for pulling images.
```

Comment:
unfortunately, I think we need a release to buffer this change. We shouldn't make such a quick change to crio.conf. We can mark as deprecated this release and drop next

Reply:
The change for insecure_registries will be reverted.  Dropping registries is required for supporting short-name aliasing without bigger changes to c/image.  @rhatdan @mrunalp @mtrmac and I found that the best way forward, but please raise your voice if you have concerns.

Reply:
another option is we can mark as deprecated mid-1.20. since we only just release 1.20.0, and most people won't move there for a while anyway, I think that would be relatively low risk

Reply:
So we're good merge into the main branch and would add deprecation warnings to the 1.20 (and 1.21?) branches?

Reply:
main branch == 1.21 (it auto backports). so yeah if we manually add deprecation warnings in 1.20 I'm okay with dropping here